_posts/2017/2017-16-01-testing-async-interactions.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: "Testing asynchronous interactions in distributed systems"
```

Comment:
@otrosien @BirgitBader I am always open to improved headlines - something like 

"How to test event-based microservices"
"How to apply contract-based testing in event-heavy systems"